cypress/integration/journalistCanCreateArticles.feature.js

```diff
@@ -1,96 +1,98 @@
 /* eslint-disable no-undef */
-describe("Journalist can create articles", () => {
+describe('Journalist can create articles', () => {
   beforeEach(() => {
     cy.server();
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/auth/sign_in",
-      response: "fixture:journalist_can_login.json",
+      method: 'POST',
+      url: 'http://localhost:3000/api/auth/sign_in',
+      response: 'fixture:journalist_can_login.json',
       headers: {
-        uid: "journalist@mail.com",
+        uid: 'journalist@mail.com',
       },
     });
     cy.route({
-      method: "GET",
-      url: "http://localhost:3000/api/auth/validate_token**",
-      response: "fixture:journalist_can_login.json",
+      method: 'GET',
+      url: 'http://localhost:3000/api/auth/validate_token**',
+      response: 'fixture:journalist_can_login.json',
     });
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/articles",
-      response: { message: "Your article was successfully created!" },
+      method: 'POST',
+      url: 'http://localhost:3000/api/articles',
+      response: { message: 'Your article was successfully created!' },
     });
-    cy.visit("/");
+    cy.visit('/');
   });
 
-  it("successfully", () => {
+  it('successfully', () => {
+    const title = 'Covid vaccine is found';
     cy.get("[data-cy='login-form']").within(() => {
-      cy.get("[data-cy='email']").type("journalist@mail.com");
-      cy.get("[data-cy='password']").type("password");
-      cy.get("[data-cy='submit-btn']").contains("Submit").click();
+      cy.get("[data-cy='email']").type('journalist@mail.com');
+      cy.get("[data-cy='password']").type('password');
+      cy.get("[data-cy='submit-btn']").contains('Submit').click();
     });
     cy.get('[data-cy="create-article-form"]').within(() => {
-      cy.get('[data-cy="input-title"]').type("Covid vaccine is found");
+      cy.get('[data-cy="input-title"]').type(title);
       cy.get('[data-cy="input-sub-title"]').type(
-        "Each person should receive two doses of vaccine"
+        'Each person should receive two doses of vaccine'
       );
       cy.get('[data-cy="input-content"]').type(
-        "Russia has distributed vaccines months ago"
+        'Russia has distributed vaccines months ago'
       );
+      cy.get('[name="file_input"]').attachFile('image.png');
```

Comment:
Is it possible to have a proper image that shows correctly so we really can see that the image is found correctly?

Reply:
That is a proper image, it’s saved in the fixture file. We also have an image preview function in the form input; the test shows this all happening

Reply:
The image looks broken when running the test,  that's why we were wondering

Reply:
ohh that's strange. I tried running on mine and it was okay. Thanks.

Reply:
Must run only the successful test to see it displayed properly :)